docs/_includes/_joiningBlocks/footer/10-menu.njk

```diff
@@ -0,0 +1,40 @@
+            </g>
+            <path d="M85.852 0H1.147C.239 0-.5.744-.5 1.658v28.969c0 .207.167.373.372.373h87.256c.205 0 .372-.166.372-.373V1.658C87.5.744 86.762 0 85.852 0zM1.147.75h84.705c.498 0 .902.406.902.908v28.557H.245v-8.426V1.658c0-.501.405-.908.902-.908z"/>
+            <ellipse cx="14.156" cy="15.661" fill="#FFF" rx="11.004" ry="11.076"/>
+            <path d="M14.22 8.746c-3.862 0-4.834 3.669-4.834 6.779 0 3.111.971 6.779 4.834 6.779 3.863 0 4.834-3.67 4.834-6.779 0-3.111-.971-6.779-4.834-6.779zm0 2.555c.157 0 .3.024.435.06.278.24.414.573.147 1.038l-2.572 4.76c-.079-.603-.091-1.195-.091-1.634 0-1.37.094-4.224 2.081-4.224zm1.926 2.193c.137.731.155 1.493.155 2.03 0 1.37-.094 4.223-2.08 4.223-.156 0-.301-.017-.435-.049-.025-.01-.049-.019-.074-.025-.04-.012-.084-.024-.122-.041-.442-.188-.721-.531-.319-1.139l2.875-4.999z"/>
+            <path d="M14.195 3.748c-3.245 0-5.98 1.137-8.21 3.422-1.128 1.135-1.99 2.431-2.589 3.876-.585 1.43-.876 2.921-.876 4.478 0 1.57.291 3.062.876 4.479s1.434 2.69 2.548 3.826c1.128 1.121 2.395 1.985 3.802 2.588 1.421.59 2.903.884 4.449.884 1.547 0 3.05-.304 4.499-.907 1.448-.604 2.74-1.471 3.883-2.605 1.101-1.078 1.934-2.317 2.49-3.719.571-1.415.853-2.932.853-4.544 0-1.598-.281-3.112-.852-4.528-.571-1.429-1.407-2.693-2.507-3.801-2.298-2.302-5.092-3.449-8.366-3.449zm.049 2.119c2.646 0 4.904.944 6.784 2.836.906.912 1.6 1.954 2.073 3.119.473 1.164.713 2.398.713 3.703 0 2.707-.92 4.952-2.744 6.746-.948.927-2.012 1.638-3.196 2.128-1.17.489-2.375.732-3.63.732-1.268 0-2.481-.239-3.638-.717-1.156-.489-2.193-1.191-3.113-2.104-.92-.925-1.629-1.97-2.13-3.135-.487-1.178-.738-2.391-.738-3.653 0-1.276.251-2.497.738-3.662.501-1.178 1.211-2.235 2.13-3.175 1.824-1.876 4.077-2.818 6.751-2.818z"/>
+          </svg>
+        </a>
+
+        <a href="https://www.netlify.com">
+          <img src="https://www.netlify.com/img/global/badges/netlify-dark.svg" alt="Deploys by Netlify" height="65px" width="100px">
```

Comment:
```suggestion
          <img src="https://www.netlify.com/img/global/badges/netlify-dark.svg" alt="Deploys by Netlify" height="65" width="100">
```

Height and width attributes only accept numbers without units; it defaults to pixels.